### PR TITLE
Legacy: support tron network

### PR DESCRIPTION
**Pull request by lyxyx**

### CODE REVIEW

File: legacy/firmware/protob/messages-tron.proto [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+/**
+ * Request: Ask device to sign message
+ * @next TronMessageSignature
+ * @next Failure
+ */
+message TronSignMessage {
+    repeated uint32 address_n = 1;                          // BIP-32 path to derive the key from master node
+    required bytes message = 2;                             // message to be signed
+    required bool do_hash = 3;                              // message need do hash?
```
Comment (revan-zhang) [inline legacy/firmware/protob/messages-tron.proto]: do_hash 最好还是去掉，跳过补充 header 有风险，固件层面可以不留这个口子

File: legacy/firmware/protob/messages-tron.proto [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    required bool is_text = 4;                              // message is text?
+    optional string prompt = 5;                             // prompt for the message to be signed
```
Comment (revan-zhang) [inline legacy/firmware/protob/messages-tron.proto]: 之前调试的时候没有用到？最好没有用到的可以一并去掉

**Review by somebodyLi — DISMISSED** [review]

File: legacy/firmware/protob/messages-tron.proto (hunk 2 above)
Comment (lyxyx) [reply legacy/firmware/protob/messages-tron.proto]: [5b3f0a3](https://github.com/OneKeyHQ/firmware/pull/67/commits/5b3f0a370a3c104986fff6957094d6181aa6a3ab)

File: legacy/firmware/protob/messages-tron.proto (hunk 1 above)
Comment (lyxyx) [reply legacy/firmware/protob/messages-tron.proto]: [5b3f0a3](https://github.com/OneKeyHQ/firmware/pull/67/commits/5b3f0a370a3c104986fff6957094d6181aa6a3ab)

File: legacy/firmware/protob/messages-tron.proto (hunk 2 above)
Comment (lyxyx) [reply legacy/firmware/protob/messages-tron.proto]: [a76ab88](https://github.com/OneKeyHQ/firmware/pull/67/commits/a76ab887370a8d27b9e5bcedd148135fb451d9b6)

File: legacy/firmware/protob/messages-tron.proto (hunk 1 above)
Comment (lyxyx) [reply legacy/firmware/protob/messages-tron.proto]: [a76ab88](https://github.com/OneKeyHQ/firmware/pull/67/commits/a76ab887370a8d27b9e5bcedd148135fb451d9b6)

**Review by revan-zhang — DISMISSED** [review]

**Review by somebodyLi — DISMISSED** [review]

**Review by lihuanhuan — DISMISSED** [review]

**Review by somebodyLi — DISMISSED** [review]

**Review by somebodyLi — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - cron: '30 19 * * 0,1,2,3,4'
+  workflow_dispatch:
+
+jobs:
+  build_firmware:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2.3.4
```
Comment (revan-zhang) [inline .github/workflows/build.yml]: action 需要提到主分支，checkout bixin_dev 分支，tron 的代码还是单独 PR 合入吧

Comment (lyxyx) [reply .github/workflows/build.yml]: https://github.com/OneKeyHQ/firmware/commit/2704d0eafb308d7865cacebf88132518963d5b12

**Review by revan-zhang — APPROVED** [review]

**Review by somebodyLi — APPROVED** [review]

**Pull request merged**